homeassistant/components/cloud/client.py

```diff
@@ -68,6 +69,7 @@ def websession(self) -> aiohttp.ClientSession:
     @property
     def aiohttp_runner(self) -> aiohttp.web.AppRunner:
         """Return client webinterface aiohttp application."""
+        assert self._hass.http.runner is not None
```

Comment:
this can be None, so type should also be `| None`. It's guarded here https://github.com/NabuCasa/hass-nabucasa/blob/master/hass_nabucasa/remote.py#L213-L215